src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -720,6 +720,25 @@ final case class ProcessApiServiceImpl(
       suspendDescriptorResponse[Unit](operationLabel)(_ => suspendDescriptor204)
     }
   }
+
+  override def getEServiceConsumers(offset: Int, limit: Int, eServiceId: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerEServiceConsumers: ToEntityMarshaller[EServiceConsumers],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
+  ): Route = authorize(ADMIN_ROLE, API_ROLE, SECURITY_ROLE, M2M_ROLE) {
```

Comment:
FYI as per a recente story, all `get` endpoints should now on be accessed also by support role
```suggestion
  ): Route = authorize(ADMIN_ROLE, API_ROLE, SECURITY_ROLE, M2M_ROLE, SUPPORT_ROLE) {
```